DigitalLearningSolutions.Data/Services/PasswordResetService.cs

```diff
@@ -171,15 +193,24 @@ string baseUrl
             emailService.ScheduleEmails(emails, addedByProcess, deliveryDate);
         }
 
+        private string GenerateResetPasswordHash(UserAccount user)
+        {
+            return GenerateResetPasswordHash(user.Id);
+        }
+
         private string GenerateResetPasswordHash(User user)
+        {
+            return GenerateResetPasswordHash(user.Id);
+        }

```

Comment:
Can you leave a TODO here that this is the incorrect ID to be passing into this method. And make sure the journeys that this is used in either have tickets, or are noted in HEEDLS-887 to come back to.

Reply:
Done